src/app/common/models/courses.model.ts

```diff
@@ -0,0 +1,7 @@
+export class Courses {
+  Id: string;
+  Title: string;
```

Comment:
Please, don't use capital letters in naming of keys, especially in combination with regular letters. Everything should be consistent.

Reply:
Updated